src/video_core/gpu.cpp

```diff
@@ -502,8 +503,15 @@ struct GPU::Impl {
         case BufferMethods::SemaphoreAddressHigh:
         case BufferMethods::SemaphoreAddressLow:
         case BufferMethods::SemaphoreSequence:
-        case BufferMethods::UnkCacheFlush:
-        case BufferMethods::WrcacheFlush:
+            break;
+        case BufferMethods::UnkCacheFlush: {
+            rasterizer->SyncGuestHost();
+            break;
+        }
```

Comment:
Unneeded curly braces
```suggestion
        case BufferMethods::UnkCacheFlush:
            rasterizer->SyncGuestHost();
            break;
```